contracts/its-aleo-translator/src/aleo/token_id_conversion.rs

```diff
@@ -0,0 +1,61 @@
+use std::ops::Index;
```

Comment:
This implementation is copied form `axelar-aleo` repo because ideally it should only leave here

Reply:
Applied at e626c406d1a183b64858966c7b1151e2d4525b56